tests/util.py

```diff
@@ -24,3 +25,11 @@
     'username': test_username,
     'password': test_password,
 }
+
+
+try:
+    import_module('lz4.frame')
```

Comment:
this seems to be unused?

Reply:
It is used by the tests for SafeDirectoryHandler. It uses built-in modules to decompress .gz, .bz2 and .xz, and if available, the lz4.frame module to decompress .lz4 files. 
Because .lz4 support is optional we only test this when it *is* available. 

BTW, we could also just add lz4 as a dependency, your call.